apps/web/src/views/Mev/hooks/index.ts

```diff
@@ -12,24 +12,17 @@ async function checkWalletSupportAddEthereumChain(walletClient: WalletClient) {
     await walletClient.request({
       method: 'wallet_addEthereumChain',
       // @ts-ignore
-      params: [
-        // mock data without key params nativeCurrency
-        {
-          chainId: '0x38', // Chain ID in hexadecimal (56 for Binance Smart Chain)
-          chainName: 'PancakeSwap MEV Guard',
-          rpcUrls: ['https://bscrpc.pancakeswap.finance'], // PancakeSwap MEV RPC}
-        },
-      ],
+      params: [{}],
```

Comment:
it is not follow the standard EIP https://eips.ethereum.org/EIPS/eip-3085, we cannot use the return value to guarantee if it support
<img width="1055" alt="image" src="https://github.com/user-attachments/assets/f4e0b425-bb0f-4a6a-bda8-515f6a8dd5c4">